### PR TITLE
mv bors.toml into .github

### PR DESCRIPTION
**Pull request by japaric**
simplify .github/workflots/ci.yml

### CODE REVIEW

File: .github/bors.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 delete_merged_branches = true
-status = ["ci"]
\ No newline at end of file
+status = ["test"]
```
Comment (jonas-schievink) [inline .github/bors.toml:2]:
```suggestion
status = [
    "test (stable, ubuntu-latest)",
    "test (stable, macOS-latest)",
    "test (stable, windows-latest)",
    "test (nightly, ubuntu-latest)",
    "test (nightly, macOS-latest)",
    "test (nightly, windows-latest)",
]
```

Comment (japaric) [reply .github/bors.toml:2]: would be nice to not need to list all the matrix combinations here. I'll add a note to the `ci.yml` stating that this other file needs to be updated if the matrix is changed (or a new job is added).

**Pull request merged**